### PR TITLE
Add recipe for r-glmtrans

### PR DESCRIPTION
**Pull request by mlondschien**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- x ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mlondschien** [comment]
Any idea what's going wrong here @conda-forge/r ?

**Comment by danielnachun** [comment]
It appears you have only committed the `meta.yaml` file for your recipe, but you also need to add `build.sh` and `bld.bat`.  Otherwise Conda doesn’t know how to actually build the recipe.

**Comment by mlondschien** [comment]
The failing osx build appears to be unrelated.

**Comment by danielnachun** [comment]
Indeed that seems to be a transient network issue.  Can you restart the CI by closing and reopening the PR?

**Comment by mlondschien** [comment]
@danielnachun can we merge this?

File: recipes/r-glmtrans/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    - conda-forge/r
+    - mlondschien
+
+# Package: glmtrans
+# Title: Search Contributed R Packages, Sort by Package
+# Version: 2.0-0
+# Author: Ye Tian [aut, cre], Yang Feng [aut]
+# Maintainer: Ye Tian <ye.t at columbia.edu>
+# Description: Transfer Learning under Regularized Generalized Linear Models
+# License: GPL (>= 2)
```
Comment (mfansler) [inline recipes/r-glmtrans/meta.yaml:67]: Where is this from? [CRAN version shows `GPL-2`](https://github.com/cran/glmtrans/blob/27cfd76926c7ff481b64ce82ad56371e15673400/DESCRIPTION#L10).

File: recipes/r-glmtrans/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    - r-formatr
+
+test:
+  commands:
+    - $R -e "library('glmtrans')"           # [not win]
+    - "\"%R%\" -e \"library('glmtrans')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=glmtrans
+  license: GPL-2.0-or-later
```
Comment (mfansler) [inline recipes/r-glmtrans/meta.yaml:50]: [CRAN shows `GPL-2`](https://github.com/cran/glmtrans/blob/27cfd76926c7ff481b64ce82ad56371e15673400/DESCRIPTION#L10), so

```suggestion
  license: GPL-2.0-only
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Mostly there, but some small corrections. Please use `conda_r_skeleton_helper` to generate CRAN skeletons.

File: recipes/r-glmtrans/meta.yaml (hunk 1 above)
Comment (mlondschien) [inline recipes/r-glmtrans/meta.yaml:67]:
```suggestion
# License: GPL-2
```

**Review by mfansler — APPROVED** [review]

**Pull request merged**